index.html

```diff
@@ -560,7 +560,9 @@ <h3>The <dfn><code>pointercancel</code> event</dfn></h3>
                 <p>A user agent MUST <a href="#firing-events-using-the-pointerevent-interface">fire a pointer event</a> named <code>pointercancel</code> in the following circumstances:</p>
                 <ul>
                     <li>The user agent has determined that a pointer is unlikely to continue to produce events (for example, because of a hardware event).</li>
-                    <li>After having fired the <code>pointerdown</code> event, if the pointer is subsequently used to manipulate the page viewport (e.g. panning or zooming).</li>
+                    <li>After having fired the <code>pointerdown</code> event, if the pointer is subsequently used to manipulate the page viewport (e.g. panning or zooming).
+                     <div class="note">User agents can use multiple pointers for a particular action like scrolling or zooming.
+                     In that case they can cancel all the pointers they deem necessary.</div></li>
```

Comment:
Nit: the indentation looks odd around here.  Please unindent by 1 char here and Lines 567-570.

Reply:
Just realized that the diff viewer in github doesn't isolate whitespace changes.  So let's not change whitespaces here.